tools/replay/unlogger.py

```diff
@@ -114,7 +114,6 @@ def _send_logs(self, data_socket):
           print("FRAME(%d) LAG -- %.2f ms" % (frame_id, fr_time*1000.0))
 
         if img is not None:
-          img = img[:, :, ::-1] # Convert RGB to BGR, which is what the camera outputs
```

Comment:
Why change this?

If I unlog a route using ./unlogger "<route name", straight from the API the colors are messed up.

Reply:
Ah, unfortunately my Panda dongle is "banned" from my.comma.ai for broken data uploads and i never got around to unbanning it.

As for why I removed that code, downloaded files are already the correct color palette. I'll make a check to see if its coming from my.comma.ai directly, and do the conversion rather than outright removal.

Who should I ping for removal of my dongle so I can test it via my.comma.ai before pushing the commit?

Reply:
Reverted that change, Was really more due to the new pipeline we are using with webcams and me testing it on that branch. sorry about that!